packages/jest-mock-apollo/README.md

```diff
@@ -10,16 +10,24 @@ $ yarn add @shopify/jest-mock-apollo
 
 ## Setup
 
-This package provides a method, `registerSchema`, which should be called in the jest `setup` file. For example:
+This package provides a factory class, which should be instantiated in a `utilities` file for your project. For example:
 
 ```ts
-registerSchema({
-  schemaBuildPath: './fixtures/schema.graphql',
-  unionOrIntersectionTypes,
+// tests/utilities
+const schemaSrc = fs.readFileSync(
+  path.resolve(__dirname, '../build/schema.graphql'),
+);
+const graphQLClientFactory = new GraphQLClientFactory({
```

Comment:
Why expose this as a class? Why not just export a factory function that just returns a `createGraphQLClient` function? Then the user wouldn't need to bother with pulling off `create` and we can remove one step from the usage guide.

I also like shipping an example that works out of the box, so we could just inline a simple schema for this.

```ts
// tests/utilities/graphql.ts

import configureMockApollo from '@shopify/jest-mock-apollo';

// You likely want to import this from somewhere  in real production code
const schema = `
  type Product {
    id: Int!
    title: String
  }

  type Query {
    products: [Product]
  }
`;

export const createGraphQLClient = configureMockApollo(schema);
```